kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/data/DataServiceImpl.java

```diff
@@ -67,6 +67,8 @@
 public class DataServiceImpl implements DataService, DataTransportListener, ConfigurableComponent,
         CloudConnectionStatusComponent, CriticalComponent, AutoConnectStrategy.ConnectionManager {
 
+    private static final int RECONNECTION_MIN_DELAY = 5;
```

Comment:
Does the workaround still work if we set this to 1 second? In this way we can be still consistent with the description of the connect.retry-interval configuration parameter, which states that the minimum value is 1 second.